site2/docs/schema-evolution-compatibility.md

```diff
@@ -6,29 +6,21 @@ sidebar_label: "Schema evolution and compatibility"
 
 Normally, schemas do not stay the same over a long period of time. Instead, they undergo evolutions to satisfy new needs. 
 
-This chapter examines how Pulsar schema evolves and what Pulsar schema compatibility check strategies are.
+This chapter introduces how Pulsar schema evolves and what compatibility check strategies it adopts.
 
 ## Schema evolution
```

Comment:
can we move this to `Understand Schema`? It feels awkward to put it here, the user must have enough context to understand it

Reply:
Yes, will be done in the next move. Leaving it in this PR just to track the changes made to it.